tests/test_json.py

```diff
@@ -0,0 +1,137 @@
+# coding: utf-8
+# Licensed under the Apache License: http://www.apache.org/licenses/LICENSE-2.0
+# For details: https://github.com/nedbat/coveragepy/blob/master/NOTICE.txt
+
+"""Test json-based summary reporting for coverage.py"""
+from datetime import datetime
+import json
+import os
+
+import coverage
+from coverage.misc import isolate_module
+from tests.coveragetest import UsingModulesMixin, CoverageTest
+
+os = isolate_module(os)
+
+
+class JsonReportTest(UsingModulesMixin, CoverageTest):
+    def _assert_expected_json_report(self, cov, expected_result):
```

Comment:
If you run "make lint", it will point out where you are missing docstrings.

Reply:
Should be set now